test/OpenSTUtility.js

```diff
@@ -78,6 +77,15 @@ const STPrime = artifacts.require("./STPrime.sol");
 /// 	fails to redeem when nonce is not > previously
 /// 	successfully redeems
 ///
+/// ProcessRedeeming
+///		BrandedToken
+/// 		fails to process if redemptionIntentHash is empty
+/// 		fails to process if msg.sender != redeemer
```

Comment:
this will be changed, like I did for ProcessStaking with the fallback option;  Good for this PR; will then complete in #26 where we implement the fallback